search/retrieval-augmented-generation/examples/rag_google_documentation.ipynb

```diff
@@ -0,0 +1,805 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    def get_similarity_fn(query_vector):
</pre><p>Does this need to have a wrapper function? Can you just pass the function directly to pandas?</p><p> </p><p>Same with <code>make_embed_text_fn()</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/240/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
`get_similarity_fn` needs to because it needs to wrap the search vector, can't parameterize it, I removed the wrapper from `make_embed_text_fn`